src/track/keyfactory.cpp

```diff
@@ -51,6 +57,23 @@ Keys KeyFactory::makeBasicKeysKeepText(
     key_map.set_global_key_text(global_key_text.toStdString());
     mixxx::track::io::key::ChromaticKey global_key = KeyUtils::guessKeyFromText(global_key_text);
     key_map.set_global_key(global_key);
+    // RapidEvolution writes tuning offset in cents after the key text, e.g. "A#m +50".
+    // Preserve that as tuning_frequency_hz if present.
+    const auto offsetMatch = s_offsetRegex.match(global_key_text);
+    if (offsetMatch.hasMatch()) {
+        bool ok = false;
+        const int cents = offsetMatch.captured(1).toInt(&ok);
+        // Validate cents range: +/- 1 octave (1200 cents) is reasonable
+        if (ok && cents >= -1200 && cents <= 1200) {
+            const double tuningHz = 440.0 * std::pow(2.0, static_cast<double>(cents) / 1200.0);
+            const int tuningRounded = static_cast<int>(std::lround(tuningHz));
```

Comment:
Avoid rounding. 
Just use a double. We use double for key infos in the engine anyway.